packages/kbn-ecs/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": "../../tsconfig.bazel.json",
+  "compilerOptions": {
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "outDir": "target_types",
+    "types": [
+      "jest",
+      "node",
+      "react"
```

Comment:
I might be wrong on this :P, but I dont think you need react in this array since this is just types. likewise, you can also take out the `"**/*.tsx",`